authority/voting/client/client.go

```diff
@@ -326,61 +321,65 @@ func (c *Client) Get(ctx context.Context, epoch uint64) (*pki.Document, []byte,
 	defer close(doneCh)
 
 	// Dispatch the get_consensus command.
-	resp, err := c.pool.fetchConsensus(ctx, linkKey, epoch)
-	if err != nil {
-		return nil, nil, err
-	}
+	// Try to fetch a consensus up to threshold number of times
+	for i := 0; i < len(c.cfg.Authorities)/2; i++ {
+		resp, err := c.pool.fetchConsensus(ctx, linkKey, epoch)
+		if err != nil {
+			return nil, nil, err
```

Comment:
is this retried somewhere else, or should we just log + `continue` here if connecting to the auth failed?

Reply:
oops. I've refactored this again, see commit below.